src/styles/blocks/card.css

```diff
@@ -0,0 +1,101 @@
+
+.card {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  background-color: var(--cont-color);
+  border: 1px solid var(--button_hover-color);
+}
+
+.mac {
```

Comment:
The class name 'mac' should be a more descriptive BEM naming convention, such as 'card__image'.